packages/nextra-theme/package.json

```diff
@@ -51,7 +51,7 @@
     "@types/react-dom": "^18.3.0",
     "next": "^14.2.5",
     "next-seo": "^6.5.0",
-    "nextra": "^2.13.4",
+    "nextra": "^3.0.0-alpha.30",
```

Comment:
Oh, still alpha. I thought it was released. Seems pretty stable though. 😁

Reply:
Yes, I need to update own `https://nextra.site` docs to include v3 changes before cutting a release, hope to do soon